model/config.go

```diff
@@ -3617,7 +3617,7 @@ func (s *LocalizationSettings) isValid() *AppError {
 	return nil
 }
 
-func (s *MessageExportSettings) isValid(fs FileSettings) *AppError {
+func (s *MessageExportSettings) isValid() *AppError {
```

Comment:
In theory, this is breaking "backward compatibility" for people using the model package (for example, people using the go client library, probably we are not break anybodies code with this, but is a breaking change.)

Reply:
This is not an exported method. So it's not a breaking change.

Reply:
Ohh, you are right. :)